app/models/person.rb

```diff
@@ -288,6 +288,10 @@ def first_name_with_initial
   # Deprecated: This is view logic (how to display name) and thus should not be in model layer
   # Consider using PersonViewUtils
   def name(community_or_display_type=nil)
+    # TODO Make `community_or_display_type` param mandatory.
+    # The custom error is here in order to get better error message while, but it can be removed in the future.
+    raise ArgumentError.new("Missing community or display type. Can not know how to format the name") if community_or_display_type.nil?
+
```

Comment:
The idea was to remove it a bit later. However, I can definitely see myself forgetting to remove this, and it will stay there forever. So, I guess the safest bet is just remove it immediately. I'll do that.